src/chart/heatmap/HeatmapView.ts

```diff
@@ -225,6 +228,17 @@ class HeatmapView extends ChartView {
         for (let idx = start; idx < end; idx++) {
             let rect;
             const style = data.getItemVisual(idx, 'style');
+            const itemModel = data.getItemModel<HeatmapDataItemOption>(idx);
+
+            // borderRadius must be set before the Rect draw.
```

Comment:
It's not necessary to fetch the borderRadius before creating `Rect`. You can still change the shape after `Rect` created. I will describe it bellow.